app/views/rubygems/show.html.erb

```diff
@@ -115,6 +115,17 @@
         </i>
       </h2>
 
+      <h2 class="gem__ruby-version__heading t-list__heading">
+        <%= t '.native_extensions' %>:
+        <i class="gem__ruby-version">
+          <% if !@latest_version.extensions.nil? %>
```

Comment:
I'd be interested in knowing that a gem has no native extensions when shopping around.